pkg/util/status/build_status.go

```diff
@@ -89,13 +89,6 @@ const (
 	// range of failures.
 	ReasonMessageGenericS2iBuildFailed api.StepFailureMessage = "Generic S2I Build failure - check S2I logs for details"
 
-	// ReasonTarSourceFailed is the failure reason associated with a failure to
-	// tar the current source.
-	ReasonTarSourceFailed api.StepFailureReason = "TarSourceFailed"
-	// ReasonMessageTarSourceFailed is the message associated with a failure to
-	// tar the current source.
-	ReasonMessageTarSourceFailed api.StepFailureMessage = "Failed to tar source files"
-
```

Comment:
Happy to see this go away :-)

@PI-Victor are these values already somehow depended on in Origin code?